XTS/localjunit/WSTX11-interop/src/main/java/com/jboss/transaction/txinterop/interop/states/BAInteropUnsolicitedCompleteState.java

```diff
@@ -54,7 +49,7 @@ public BAInteropUnsolicitedCompleteState()
      */
     public ContentHandler getHandler(final ContentHandler nextHandler)
     {
-	return (replaced ? null : new RegistrationHandler(nextHandler)) ;
+	return new RegistrationHandler(nextHandler);
```

Comment:
I agree with the change in principal but it would be good to understand what replaced might have been intended to do.